callgraph/core.py

```diff
@@ -24,15 +26,20 @@ def AddCommand(self, command):
 
     def __repr__(self):
         return "[{0} (terminating: {1}, noop: {2}, commands: {3})] {4}".format(self.number, self.terminating, self.noop, self.commands, self.text)
-    
+
     def __eq__(self, other):
         return other is not None and self.number == other.number and self.text == other.text and self.terminating == other.terminating
 
 # Connection between two nodes.
-# dst is the name of the target node, kind is the type of connection, line_number is the line where the call/goto happens.
+# dst is the name of the target node, kind is the type of connection,
+# line_number is the line where the call/goto happens.
+
+

```

Comment:
Could you remove those spaces between the comment about the namedtuple and the definition of the namedtuple itself?

Reply:
Yes, I have removed the spaces.